components/kubeflow/deployer/src/deploy.sh

```diff
@@ -95,6 +100,14 @@ echo "Generating the TF Serving config..."
 ks generate tf-serving server --name="${SERVER_NAME}"
 ks param set server modelPath "${MODEL_PATH}/export/export"
 
+# support local storage to deploy tf-serving.
+if [ ! -z "${PVC_NAME}" ];then
```

Comment:
Can it be [ -n "${PVC_NAME}" ] since this is more readable

Reply:
@gaoning777 Thanks for your comments. Updated.